### PR TITLE
Works-aboutMyself-section

### PR DESCRIPTION
**Pull request by ElShaer398**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+          <li>css</li>
+          <li>javaScipt</li>
+        </ul>
+      </div>
+      <a href="#projects" class="projects-link">See Project</a>
+    </div>
+  </section>
+  <section class="about-myself-section">
+    <h1 class="about-myself">About<br>Myself</h1>
+    <div class="about-myself-text">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</div>
```
Comment (ElsonOtake) [inline index.html:113]: - [ ] I think it's better to use a `<p>` tag for this paragraph text. It's semantically correct and creates a more professional code.

File: index.html [hunk 2]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+    <div class="connect">LET'S CONNECT</div>
```
Comment (ElsonOtake) [inline index.html:114]: - [ ] I think it's better to use a header tag like `<h2>`, `<h3>`, ... for this header text. It's semantically correct and allows search engines to find information in your code easily.

File: index.html [hunk 3]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+    <ul class="social-icons" id="social-icons-2">
+      <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+    </ul>
+    <a href="#resume" class="projects-link" id="resume">Get my resume</a>
+    <div class="languages-block">
```
Comment (ElsonOtake) [inline index.html:122]: - [ ] I think it's better to use the "languages-block" as a list for semantical reasons. Use `<ul>` to group the Languages, Frameworks, and Skills. Put each category in a `<li>` tag. Your code will look more professional.

File: main.css [hunk 4]
```diff
@@ -65,3 +65,181 @@ header {
...
+  text-decoration: none;
+  border-style: solid;
+  border-width: 1px;
+  border-radius: 8px;
+  border-color: #396df2;
+  padding: 8px 12px;
+  margin-top: 13px;
+}
+
+.about-myself-section {
```
Comment (ElsonOtake) [inline main.css:164]: - [ ] Please use flexbox to align this section. It's important to follow the project requirement.

File: index.html [hunk 5]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+      <span class="languages">Languages</span>
```
Comment (ElsonOtake) [inline index.html:123]: - [ ] Kindly add that arrow down as in the template after Languages. Stick to the template is a project requirement.

Figma template

Screen Shot 2022-11-05 at 4 10 22 PM

Your design

Screen Shot 2022-11-05 at 4 10 39 PM

File: index.html [hunk 6]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+      <ul>
+        <li class="li-1">javaScipt</li>
+        <li class="li-2">HTML</li>
+        <li class="li-3">CSS</li>
```
Comment (ElsonOtake) [inline index.html:127]: - [ ] Please align your languages centered vertically as in the template. Stick to the template is a project requirement.

Figma template

Screen Shot 2022-11-05 at 4 10 22 PM

Your design

Screen Shot 2022-11-05 at 4 10 39 PM

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @ElShaer398,

Good job so far! 🎉 🎉  Nice work by the way!! 🏆 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please create a descriptive README file. It's the first thing that recruiters and hiring managers will check when look for your work. Use the provided [template](https://github.com/microverseinc/readme-template) and make the changes to customize to your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+  <section class="cards-section">
+    <div class="card" id="card-1">
+      <img src="images/photo-1.png" alt="profile-1">
+      <h2 class="profile-name">Tonic</h2>
+      <div>
+        <span class="canopy">CANOPY</span>
+        <ul class="job">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
```
[the remark this answers is not in the document]
Comment (ElShaer398) [reply index.html:38]: When I make the list stack horizontally the dots disappear and I tried to solve it but I couldn't.

File: index.html [hunk 8]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+      </div>
+      <div class="primary-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</div>
+      <div class="skills">
+        <ul>
+          <li>html</li>
+          <li>css</li>
+          <li>javaScipt</li>
+        </ul>
+      </div>
+      <a href="#projects" class="projects-link">See Project</a>
```
[the remark this answers is not in the document]
Comment (ElShaer398) [reply index.html:48]: Where are these styles, I can't find hover styles.

[the remark this answers is not in the document]
Comment (bienvenuushindi) [reply index.html:48]: :red_circle:   @ElShaer398  You can find the different styles from the Figma Design

Buttons styles :arrow_down: 
buttons

Please make sure you pick the accurate state for the hover effect.`(color is white and background is #6070FF;)`

File: index.html [hunk 9]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
```
[the remark this answers is not in the document]
Comment (bienvenuushindi) [reply index.html:68]: :red_circle:   @ElShaer398  You can find the different styles from the Figma Design

Buttons styles :arrow_down: 
buttons

Please make sure you pick the accurate state for the hover effect.`(color is white and background is #6070FF;)`

File: index.html [hunk 10]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
```
[the remark this answers is not in the document]
Comment (bienvenuushindi) [reply index.html:88]: :red_circle:   @ElShaer398  You can find the different styles from the Figma Design

Buttons styles :arrow_down: 
buttons

Please make sure you pick the accurate state for the hover effect.`(color is white and background is #6070FF;)`

File: index.html [hunk 11]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
```
[the remark this answers is not in the document]
Comment (bienvenuushindi) [reply index.html:108]: :red_circle:   @ElShaer398  You can find the different styles from the Figma Design

Buttons styles :arrow_down: 
buttons

Please make sure you pick the accurate state for the hover effect.`(color is white and background is #6070FF;)`

File: index.html [hunk 12]
```diff
@@ -26,5 +26,111 @@ <h1 class="name">I'm Mohammad Glad to see you!</h1>
...
+    <h1 class="about-myself">About<br>Myself</h1>
+    <div class="about-myself-text">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</div>
+    <div class="connect">LET'S CONNECT</div>
+    <ul class="social-icons" id="social-icons-2">
+      <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+    </ul>
+    <a href="#resume" class="projects-link" id="resume">Get my resume</a>
```
[the remark this answers is not in the document]
Comment (bienvenuushindi) [reply index.html:121]: :red_circle:   @ElShaer398  You can find the different styles from the Figma Design

Buttons styles :arrow_down: 
buttons

Please make sure you pick the accurate state for the hover effect.`(color is white and background is #6070FF;)`

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @ElShaer398 ,

Good job on implementing most of the required changes!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔
- Nice Design. ✔
- 

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] Please create a descriptive README file. It's the first thing that recruiters and hiring managers will check when looking for your work. Use the provided [template](https://github.com/microverseinc/readme-template) and make the changes to customize your project.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @ElShaer398 , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**